### DISCUSSION TITLE
Welcome to i18n-collect-cli Discussions!

### DISCUSSION THREAD

**Discussion by xujiujiu** [xujiujiu/i18n-collect-cli#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EthanYQ** [comment]
词条分成cs_view和common的作用是什么

**Reply by xujiujiu** [reply]
区分通用字符和业务字符，通用字符能多次使用，cs_view里是业务字符，标记页面
